client/src/main/kotlin/io/spine/examples/chatspn/desktop/ChatsPage.kt

```diff
@@ -475,17 +506,258 @@ private fun ChatNotChosenBox() {
 }
 
 /**
- * Represents the topbar of chat content.
+ * Represents the info tab with the user profile.
+ */
+@Composable
+private fun ProfileTab(model: ChatsPageModel) {
```

Comment:
I think, profiles should belong to their own Kotlin file. 

It's weird that they are a part of chats, even though at the moment these features are somewhat related in navigation.